src/chain.ts

```diff
@@ -11,7 +11,7 @@ export type Block = {
     score: string | number;
     seal: number[][];
     stateRoot: string;
-    validatorSetRoot: string;
+    nextValidatorSetHash: string;
```

Comment:
Why did you change this line?
It is not related to the "chain_getRegularKey" RPC.

Reply:
Please create a new commit for this change.